xgoedu/__init__.py

```diff
@@ -203,6 +221,28 @@ def lcd_picture(self,filename,x=0,y=0):
         image = Image.open(path+filename)
         self.splash.paste(image,(x,y))
         self.display.ShowImage(self.splash)
+     #显示图片
+    '''
+        按窗口比例缩放图片显示；  大小为320*240 
+    '''
+    def lcd_picture_resize(self, filename, x=0, y=0):
+        path = "/home/pi/xgoPictures/" 
```

Comment:
It is a very bad idea to hardcode paths in the middle of the code.
It should be declared in the head of the source file.